docs/tempo/website/grafana-agent/tail-based-sampling.md

```diff
@@ -49,8 +49,8 @@ tempo:
         policies:
```

Comment:
- have we revendored otel to add additional features? can we add examples of tail based configs that use the additional features?
- Can we include some details about how you only need the load balancing if you have more than one collector?

Reply:
> have we revendored otel to add additional features? can we add examples of tail based configs that use the additional features?

Yes, it's on v0.30 now. Added examples for the two new policies.

> Can we include some details about how you only need the load balancing if you have more than one collector?

I think it's explained in the description, is anything missing in particular?

Reply:
> Can we include some details about how you only need the load balancing if you have more than one collector?

Maybe we should make a separate section for load balancing? This will make it easier to distinguish between how tail sampling itself works and how you get it working with multiple agent instances.

Reply:
Maybe? I don't have a strong opinion about it, so I'm fine with whatever the team thinks is better in this case.